### PR TITLE
[MINOR][SQL][DOCS] Fix some wrong default values in SQL tuning guide's AQE section

### PR DESCRIPTION
**Pull request by yaooqinn**
### What changes were proposed in this pull request?

spark.sql.adaptive.coalescePartitions.initialPartitionNum 200 -> (none)
spark.sql.adaptive.skewJoin.skewedPartitionFactor is 10 -> 5

### Why are the changes needed?

the wrong doc misguide people
### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

passing doc

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @maropu @cloud-fan thanks

**Review by maropu — APPROVED** [review]
Nice catch!

File: docs/sql-performance-tuning.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ This feature coalesces the post shuffle partitions based on the map output stati
    </tr>
    <tr>
      <td><code>spark.sql.adaptive.coalescePartitions.initialPartitionNum</code></td>
-     <td>200</td>
+     <td>(none)</td>
      <td>
        The initial number of shuffle partitions before coalescing. By default it equals to <code>spark.sql.shuffle.partitions</code>. This configuration only has an effect when <code>spark.sql.adaptive.enabled</code> and <code>spark.sql.adaptive.coalescePartitions.enabled</code> are both enabled.
```
Comment (dongjoon-hyun) [inline docs/sql-performance-tuning.md:260]: It seems that we had better change this part together: `By default` -> `If not set,`

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM (with a minor comment)

**Review by HyukjinKwon — APPROVED** [review]

**Review by cloud-fan — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master, branch-3.1 and branch-3.0.

**Pull request closed without merging**